CHANGELOG.next.md

```diff
@@ -1,63 +1,34 @@
 <!-- When adding an entry to the Changelog:
-
```

Comment:
@janniten You're accidentally overwriting the current changelog with an older format.

To fix this, you can run `git checkout CHANGELOG.next.md` and then introducing your changelog entry after that ;-)

Reply:
Fixed (I think so : ) )